Example/Tests/Tests.swift

```diff
@@ -1,27 +1,136 @@
 import XCTest
 import QBRepository
+import RealmSwift
+
+class RealmRepositoryTests: XCTestCase {
+
+    var repository: RealmRepository<QuickEmployee>?
 
-class Tests: XCTestCase {
-    
     override func setUp() {
         super.setUp()
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+        let testRealm = try! Realm(configuration: Realm.Configuration(inMemoryIdentifier: "Test"))
+        repository = RealmRepository<QuickEmployee>(realm: testRealm)
     }
     
     override func tearDown() {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        let testRealm = try! Realm(configuration: Realm.Configuration(inMemoryIdentifier: "Test"))
+        repository = RealmRepository<QuickEmployee>(realm: testRealm)
         super.tearDown()
     }
     
-    func testExample() {
-        // This is an example of a functional test case.
+    func testGetAll() {
+        addRandomMockObjects(repo: repository)
```

Comment:
Setup method is called before the invocation of each test method in the class. So this method call should only be part of the setUp method